### PR TITLE
Restoring GitHub Pages

### PR DESCRIPTION
**Pull request by dreamwhite**
Hi there,
I'm willing to fix GitHub pages integration with this repo. It would be really helpful to provide a nice looking Jekyll website that shows every single hash.

I'm actually coding a script that converts the actual `.md` files onto a JSON as already did back in 2017 by @notpeter so GitHub pages bot can easily rebuild back everything.

At the moment I managed to obtain the following output:

image

It shouldn't take that much but I gotta figure out how to properly parse [this table](https://github.com/notpeter/apple-installer-checksums/tree/master#mac-osx-installers-sha1-checksums=) as I tested only with a few strings 😅.

Basically my idea is creating a GitHub Action that when a new commit is pushed onto master branch runs a script to create the json , then pushes it onto `gh-pages` branch so GitHub Pages can rebuild everything.

Let me know your thoughts

Greetings
dreamwhite

### CODE REVIEW

**Comment by dreamwhite** [comment]
I'm achieving some progress. The algorithm for identifying version number of each hash outputs this:

```
13.0db1
12.4
12.3.1
12.3
12.2.1
12.2
12.1
12.0.1
12.0db10
12.0db9
12.0db8
12.0db7
12.0db6
12.0db5
12.0db4
12.0db3
12.0db2
12.0db1
11.6.7
11.6.6
11.6.5
11.6.4
11.6.3
11.6.2
11.6.1
11.6
11.5.2
11.5.1
11.5
11.4
11.3.1
11.3
11.2.3
11.2.2
11.2.1
11.2
11.1
11.0.1
11.0.1
11.0.1rc2
11.0.1rc
11.0.1db1
11.0db10
11.0db9
11.0db6
11.0db5
11.0db4
11.0db3
10.16db2
10.16db1
10.15.7
10.15.7
10.15.7
10.15.6
10.15.6
10.15.5
10.15.5
10.15.4
10.15.4
10.15.3
10.15.3
10.15.2
10.15.1
10.15.0
10.15.0
10.15.0
10.15gm
10.15db10
10.15db9
10.15db5
10.15db4
10.15db3
10.15db2
10.15db1
10.14.6 2019-10-03)
10.14.6 2019-09-20)
10.14.6
10.14.6
10.14.5
10.14.4
10.14.4
10.14.3
10.14.3
10.14.2
10.14.1
10.14.0
10.14db10
10.14db9
10.14db8
10.14db7
10.14db6
10.14db5
10.14db4
10.13.6
10.13.6
10.13.5
10.13.4
10.13.3
10.13.2
10.13.1
10.13.0
10.13.0
10.13gmcandidate High Sierra
10.13db9 High Sierra
10.13db8 High Sierra
10.13db7 High Sierra
10.13db6 High Sierra
10.13db5 High Sierra
10.13db4 High Sierra
10.13db3 High Sierra
10.13db2u1 High Sierra
10.13db2 High Sierra
10.13db1 High Sierra
10.12.6 Sierra
10.12.6 Sierra
10.12.5 Sierra
10.12.4 Sierra
10.12.3 Sierra
10.12.2 Sierra
10.12.2 Sierra
10.12.1 Sierra
10.12.1 Sierra
10.12.0 Sierra
10.12GM Sierra
10.12GM Sierra
10.12dp2 Sierra
10.12dp1 Sierra
10.11.6 El Capitan
10.11.6 El Capitan
10.11.6 El Capitan
10.11.6 El Capitan
10.11.5 El Capitan
10.11.4 El Capitan
10.11.3 El Capitan
10.11.2 El Capitan
10.11.1 El Capitan
10.11.0 El Capitan
10.10.5 Yosemite
10.10.5 Yosemite
10.10.5 Yosemite
10.10.4 Yosemite
10.10.3 Yosemite
10.10.3 Yosemite
10.10.2 Yosemite
10.10.1 Yosemite
10.10.0 Yosemite
10.9.5 Mavericks
10.9.5 Mavericks
10.9.1 Mavericks
10.9.0 Mavericks
10.8.5 Mountain Lion
10.8.5 Mountain Lion
10.8.2 Mountain Lion
10.8.0 Mountain Lion
10.7.5 Lion
10.6.0 Snow Leopard
10.5.0 Leopard
```

What is actually broken is the date check. Lemme explain: given `10.14.6 (18G103, 2019-10-03)` the algorithm should extract the build version, the build number and if there the build date. Actually it outputs only `10.14.6 2019-09-20)` which is not what we want

**Comment by dreamwhite** [comment]
Fixed, will proceed in the following days on filename and hashes detection. Stay tuned

**Comment by notpeter** [comment]
Excited to see someone else pick this up. It struck me as a natural extension to create a proper ghpages website with a better interface, but after many hours I gave up. Let me know when you'd like a review or if you need help with regexes, repo config, etc.

**Comment by dreamwhite** [comment]
Thank you so much! I'm actually having some issues with my local jekyll environment regarding the link to `xcode.md` but it shouldn't take that much to figure it out.
In any case it should be something like this

image

**Comment by dreamwhite** [comment]
@notpeter can you actually retry building the website?

**Comment by dreamwhite** [comment]
Good morning @notpeter any news about this PR?
